### ISSUE TITLE
'TypeError: importItems.importedAs.entries is not a function' error when using --cache, but not without the cache

### ISSUE THREAD

**Issue by philwolstenholme** [webpro-nl/knip#656]
Hi,

When I run Knip 5.17.0 (on Node 20.12.0) with the `--cache` option I get this error:

```
file:/// … /node_modules/knip/dist/index.js:186
        for (const [id, value] of importItems.importedAs.entries()) {
                                                         ^

TypeError: importItems.importedAs.entries is not a function or its return value is not iterable
    at updateImports (file:///Users/wolsph/Projects/shop-next/node_modules/knip/dist/index.js:186:58)
    at updateImported (file:///Users/wolsph/Projects/shop-next/node_modules/knip/dist/index.js:229:13)
    at setInternalImports (file:///Users/wolsph/Projects/shop-next/node_modules/knip/dist/index.js:238:13)
    at analyzeSourceFile (file:///Users/wolsph/Projects/shop-next/node_modules/knip/dist/index.js:266:13)
    at main (file:///Users/wolsph/Projects/shop-next/node_modules/knip/dist/index.js:301:17)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async run (file:///Users/wolsph/Projects/shop-next/node_modules/knip/dist/cli.js:23:73)
    at async file:///Users/wolsph/Projects/shop-next/node_modules/knip/dist/cli.js:90:1
```

If I `rm -rf node_modules/.cache` or remove the `--cache` option and re-run Knip then there is no error.

If I `console.log` `importedModule.importedAs` then it appears to be an empty object:

```
Analyzing source files...
{}
```

I had the same error when using Knip 5.16.0 so I don't think it is related to the release of 5.17.0.

**Comment by webpro** [comment]
Thanks for the report! I think it's not (de)serializing the map from/to an object (it does for sets/arrays), I can fix it later today.

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v5.17.1. See [Release 5.17.1](https://github.com/webpro-nl/knip/releases/tag/5.17.1) for release notes._

_Using Knip in a commercial project? Please consider [sponsoring me](https://github.com/sponsors/webpro)._
